src/services/changedFilesCoverage.js

```diff
@@ -16,17 +17,19 @@ const getChangedFilesCoverage = async (coverage) => {
     pullNumber
   });
 
+  const workspacePath = getWorkspacePath();
   const changedFilesCoverage = coverage.data.reduce(
     (allFiles, { file, lines }) => {
+      const filePath = workspacePath ? `${workspacePath}/${file}` : file;
```

Comment:
Ideally, would be good to use https://www.w3schools.com/nodejs/met_path_join.asp

As users can enter the workspace with and without `/`

Reply:
Also the action can be run in Linux/Windows machines theoretically

Reply:
@vitaliimelnychuk That Makes sense. using path.join() to merge the paths now.